### PR TITLE
Set default donation to Coordinape

### PR DESCRIPTION
**Pull request by levity**
default to sending 1% of GIVE to Coordinape during allocation

### TODO 

- [x] test
- [x] track usage in Mixpanel (add a property to `update_allocations` with the amount given to Coordinape)

### CODE REVIEW

**Comment by levity** [comment]
@CryptoGraffe i'd appreciate a quick sanity check on how i inserted this into the allocation logic

File: src/pages/GivePage/index.tsx [hunk 1]
```diff
@@ -357,15 +357,31 @@ const GivePage = () => {
...
+          },
+        });
+
+      return;
     }
-  }, [pendingGiftsFrom]);
+
+    // load previously-saved allocations
+    setGifts(
+      pendingGiftsFrom.reduce<typeof gifts>((map, g) => {
```
Comment (tiff0000) [inline src/pages/GivePage/index.tsx:379]: can spell out `g` as gift

**Comment by levity** [comment]
Current dependencies on/for this PR:
* main
  * **PR #2043** <a href="https://app.graphite.dev/github/pr/coordinape/coordinape/2043" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/coordinape/coordinape/2043?utm_source=stack-comment).

File: src/pages/GivePage/index.tsx (hunk 1 above)
Comment (levity) [reply src/pages/GivePage/index.tsx:379]: thanks for the suggestion. in cases like this, where we've already spelled out the name of the thing nearby and the function is small, i prefer brevity

File: src/features/auth/WalletAuthModal.tsx [hunk 2]
```diff
@@ -6,7 +6,10 @@ import { loginSupportedChainIds } from 'common-lib/constants';
 
 import { CircularProgress } from '@material-ui/core';
 
-import { EConnectorNames, WALLET_ICONS } from 'config/constants';
+import { ReactComponent as CoinbaseSVG } from 'assets/svgs/wallet/coinbase.svg';
```
Comment (crabsinger) [inline src/features/auth/WalletAuthModal.tsx:9]: @levity these changes seem out of place?

**Pull request merged**